### PR TITLE
Fixing formatting errors

### PR DESCRIPTION
**Pull request by somindatommy**
## Purpose
Fixing formatting errors in page [1].

[1] - https://is.docs.wso2.com/en/5.9.0/setup/migrating-data-publishers/

### CODE REVIEW

File: en/docs/setup/migrating-data-publishers.md [hunk 1]
```diff
@@ -3,47 +3,47 @@
 In prior versions of WSO2 Identity Server, data publishers were
 implementations of the
 [AbstractAuthenticationDataPublisher](https://github.com/wso2-extensions/identity-data-publisher-authentication/blob/master/components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/AbstractAuthenticationDataPublisher.java)
-that are invoked iteratively by the
+which were invoked iteratively by the
 [AuthnDataPublisherProxy](https://github.com/wso2-extensions/identity-data-publisher-authentication/blob/master/components/org.wso2.carbon.identity.data.publisher.application.authentication/src/main/java/org/wso2/carbon/identity/data/publisher/application/authentication/AuthnDataPublisherProxy.java)
 when a session changes, such that the data publishers send events to
-their corresponding destinations. From WSO2 Identity Server 5.8.0
+their corresponding destinations. From WSO2 Identity Server 5.9.0
```
Comment (sherenewso2) [inline en/docs/setup/migrating-data-publishers.md]: we migrated the data publishers to event handlers in 5.8.0 right? I think that's why it mentions from 5.8.0 onwards. Shall we revert this?

File: en/docs/setup/migrating-data-publishers.md [hunk 2]
```diff
@@ -3,47 +3,47 @@
...
     the event.
 
 -   These event handlers extend the
     [AbstractEventHandler](https://github.com/wso2/carbon-identity-framework/blob/master/components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/handler/AbstractEventHandler.java)
-    and override its `            handleEvent           ` method.
+    and override its `handleEvent` method.
 
-The diagrams given below illustrate the difference between the design
-approach in versions prior to WSO2 IS 5.8.0 and the new design approach.
+The following diagrams illustrates the difference between the design
```
Comment (sherenewso2) [inline en/docs/setup/migrating-data-publishers.md]: This should be corrected to the "The following diagrams **illustrate**"

File: en/docs/setup/migrating-data-publishers.md (hunk 1 above)
Comment (somindatommy) [reply en/docs/setup/migrating-data-publishers.md]: Fixed: 1325d3b997056bb2e11dd84500109c72375e078e

File: en/docs/setup/migrating-data-publishers.md (hunk 2 above)
Comment (somindatommy) [reply en/docs/setup/migrating-data-publishers.md]: Fixed: 1325d3b997056bb2e11dd84500109c72375e078e

**Pull request closed without merging**